wesnoth-units.po

```diff
@@ -2035,6 +2067,26 @@ msgid ""
 "acuity who may then be considered as candidates to lead the next generation "
 "of Dunefolk."
 msgstr ""
+"V mnoha pouštních městských státech je výběr nové generace vůdců dlouhý "
+"proces, který probíhá po mnoho let. V každé generaci je vybráno několik "
+"obzvláště talentovaných mladých lidí mezi šlechtou a je jim věnováno "
+"speciální úsilí. Jejich život je jen jídlo, spánek a trénink. Tito válečníci "
+"šermem učí žít, učí dýchat. Šerm se tam stává víc než jen jejich dovedností, "
+"stává se jejich základní identitou. Od dospívání byli po deset let "
+"intenzivním tréninkem připravováni k závěrečnému testu. Tím byla série duelů "
+"na život a na smrt mezi dvojicemi těchto bojovníků. Když se prach zvířený "
+"bojem usadil a proces byl u konce, zbylo posledních osm. Těmto mužům pak byl "
+"udělen titul Kal či souhrnně Kalai.\n"
+"\n"
+"Zatímco se mezi pouštím lidem vyžaduje, aby velcí vůdci byli mistry čepele, "
+"vládnoucí kasta musí znát mnohem víc než pouhý šerm. Po drtivém drillu "
+"obvykle následuje buď několikaleté toulání se po pouštích, nebo trénování "
+"mysli v oblasti věd. Účelem tohoto je najít cestu k osvícení, hnanou svou "
+"vlastní motivací. Ať už si vybrali jakkýkoli způsob, na konci druhého "
```

Comment:
hnaná cesta mi přijde trochu přitažená za vlasy
nestačilo by to jednodušeji?
```suggestion
"mysli v oblasti věd. Účelem toho je najít vlastní cestu k osvícení. "
"Ať už si vybrali jakkýkoli způsob, na konci druhého "
```

Reply:
Ano souhlasím. Když on je originál někdy přespříliš explicitní a já to někdy nedostatečně krouhám.